### PR TITLE
ACL for assets authoring

### PR DESCRIPTION
**Pull request by gabrielfs7**
Apply ACL for assets on authoring

https://oat-sa.atlassian.net/browse/ADF-271

- [ ] https://github.com/oat-sa/tao-core/pull/2908
- [ ] https://github.com/oat-sa/extension-tao-item/pull/470

### CODE REVIEW

**Review by shpran — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by mike-ionut-mihai-sandu-tao — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Review by voknil — APPROVED** [review]
- [x] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

File: model/MediaSource.php [hunk 1]
```diff
@@ -147,14 +158,17 @@ public function getFileInfo($link)
...
 
-        return [
-            'name' => $resource->getLabel(),
-            'uri' => self::SCHEME_NAME . tao_helpers_Uri::encode($link),
-            'mime' => (string)$mime,
-            'size' => $this->getFileManagement()->getFileSize($fileLink),
-            'alt' => $alt,
-            'link' => $fileLink
-        ];
+        return $this->getPermissionsMapper()->map(
```
[the remark this answers is not in the document]
Comment (gabrielfs7) [reply model/MediaSource.php:161]: There will be an extra query for every resource in the pagination, but only for authoring when adding assets. So other places retrieving the resources wont be affected. 

That is how tao-dac-simple already works in the ontology tree build, search results, etc. There the issue is bigger than for this part, cause we load the ontology with permission every time we load a section of tao. 

Here is isolated :)

**Review by chinnu-focaloid — APPROVED** [review]
- [x] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Comment by AntonNeviarouski** [comment]
QA verified on http://test-gabriel.playground.kitchen.it.taocloud.org:42221/

**Pull request merged**